c-pallets/sminer/src/lib.rs

```diff
@@ -587,7 +587,7 @@ pub mod pallet {
 				(DEMOCRACY_IDA).encode(),
 				DispatchTime::At(when),
 				Some((cycle, degree)),
-				60,
+				60, // const
```

Comment:
I have made changes. In order to ensure the smooth implementation of the rewards given to miners, I have selected high priority here:
https://github.com/CESSProject/cess/pull/88/files#diff-30c617cf01d014c0e882cf4c64b66e5f459de77270aa8617a71c6fc93a9a143aR593